app/src/constants.js

```diff
@@ -1,5 +1,9 @@
 const Constants = {
-    SETTING_ALPHA_UNDERSTANDS_RISKS: "ALPHA_UNDERSTANDS_RISKS"
+    KEY_FIRST_RUN_EMAIL: "firstRunEmail",
```

Comment:
Why is this value camel case unlike the others?

Reply:
The value for the `SETTING_*` key should actually be camel case.